lighthouse-core/report/templates/report-template.html

```diff
@@ -175,5 +174,27 @@ <h1 class="menu__header-title">Lighthouse</h1>
     <footer class="footer">
       Generated by <b>Lighthouse</b> {{lighthouseVersion}} on {{generatedTime}} | <a href="https://github.com/GoogleChrome/Lighthouse/issues" target="_blank">File an issue</a>
     </footer>
+    
+    <div class="config-panel js-config-panel">
```

Comment:
since this is nicely self contained, maybe we can move this into a partial, included conditionally on `reportContext`?